lib/src/chart/line_chart/line_chart_renderer.dart

```diff
@@ -90,11 +90,11 @@ class RenderLineChart extends RenderBaseChart<LineTouchResponse> {
 
   @override
   LineTouchResponse getResponseAtLocation(Offset localPosition) {
-    var touchedSpots = painter.handleTouch(
+    final touchedSpots = painter.handleTouch(
       localPosition,
       mockTestSize ?? size,
       paintHolder,
     );
-    return LineTouchResponse(touchedSpots);
+    return LineTouchResponse(touchedSpots, touchedSpots?.first);
```

Comment:
I guess we could keep `LineTouchResponse` at it was before, only containing the list of touched spots and just document the fact, that they are sorted by distance? There isn't really any need to save the list along with the first element of the list, right?

I think that's the final decision I need your thoughts on. Then I would write the tests, after which I would be finished with coding.

Reply:
I agree with removing it. Go ahead and when everything was ready let me review everything again.